robot.py

```diff
@@ -246,8 +256,23 @@ def teleopPeriodic(self):
                 FieldAngle.CARGO_FRONT.value
             )  # Reversed side of robot
 
+        if self.gamepad.getPOV() != -1:
+            speed = 0.5
+            azimuth = math.radians(-self.gamepad.getPOV())
+            if self.cargo_component.has_cargo:
+                azimuth += math.pi
+            self.chassis.set_inputs(
+                speed * math.cos(azimuth),
+                speed * math.sin(azimuth),
+                0,
+                field_oriented=False,
+            )
+
     def robotPeriodic(self):
         # super().robotPeriodic()
+        if self.timer > 0:
+            self.timer -= 0.05
+        wpilib.SmartDashboard.putNumber("timer", self.timer)
```

Comment:
Don't use SmartDashboard.putNumber() - this involves recomputing the key every time. Either use a tunable or a NetworkTableEntry.